src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+body,
+ul,
+li {
```

Comment:
Don't stick selectors together
Later it will be hard to find all the rules applied to `body` because you will need to check a lot of places